src/Login/Login.js

```diff
@@ -9,7 +9,8 @@ class Login extends Component {
     super(props) 
     this.state = {
       email: '',
-      password: ''
+      password: '',
+      loginError: ''
```

Comment:
I think this was a great change to make. Makes sense that the login has it's own error prop.